Routing/Loader/Reader/RestActionReader.php

```diff
@@ -333,39 +333,41 @@ public function read(RestRouteCollection $collection, \ReflectionMethod $method,
 
             // add route to collection
             $route = new Route(
-                $path, $defaults, $requirements, $options, $host, [], $methods, $condition
+                $path, $defaults, $requirements, $options, $host, [], $methods, $versionCondition
```

Comment:
Maybe I can rename `$condition` in line 319 to `$combinedConditions` or similar?